docs/source/getting_started/Installing.rst

```diff
@@ -237,3 +237,129 @@ se, and choose the conda file saved earlier with the Spark session configuration
     - numpy
     - pip:
         - msticpy[azure]>=2.3.1
+
+Installing for isolated or semi-isolated environments
+-----------------------------------------------------
+
+There are instances in which you may want to utilize msticpy in an isolated or semi-isolated environment. It is recommended to utilize a virtual environment and the scripts provided within the steps. Also ensure you have the correct python version installed on your host machine as well as the isolated environment. In order to find the correct python version, you can run the following:
+
+.. code-block:: powershell
+
+    python --version
+
+You can also run:
+
+.. code-block:: python
+    import sys
+    print(sys.version)
+
+Below are some examples on how to do this for certain platforms:
+
+Windows -> Isolated Windows Environment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- On your primary Windows machine with internet access create a virtual environment for the python version you want to use. 
+- Download msticpy by running the following:
+
+.. code-block:: powershell
+
+    pip download msticpy --no-deps --dest \path\to\destination
+
+- Within \path\to\destination you should see a .whl file for msticpy and the other dependencies. Some dependencies may not be .whl files, but tar.gz files. These files will need to be built into .whl files. To do this, run the following:
+
+.. code-block:: powershell
+
+    pip wheel {file.tar.gz} -w \path\to\destination
+
+or use the tool here 
```

Comment:
add (from the mstipcy repo "tools" folder)
(Just to assure people that they are not downloading some random script from who knows where) :-)

Reply:
updated